api/src/test/java/org/apache/iceberg/transforms/TestBucketing.java

```diff
@@ -32,6 +32,7 @@
 import org.apache.iceberg.relocated.com.google.common.hash.HashFunction;
 import org.apache.iceberg.relocated.com.google.common.hash.Hashing;
 import org.apache.iceberg.types.Types;
+import org.apache.iceberg.util.BucketUtil;
```

Comment:
These tests instantiated bucket transforms to call the `hash` method. That method was moved to `BucketUtil`, so this now tests those functions directly.